ChunkyDocs/docs/reference/user_interface/chunky/chunky_gui.md

```diff
@@ -0,0 +1,33 @@
+# Chunky GUI
+
+{% if extra.chunky < 2_05_00 %}
+
+The GUI of Chunky is built using JavaFX and is separated into three main resizable control panels. These panels have tabs which contain additional controls. The control panel on the left contains render controls, the control panel in the middle contains the world map view and the render preview, and the control panel on the right contains general controls. Scene management controls are at the top of the window, and information about render progress is displayed at the bottom of the window, along with a render progress bar.
+
+<div class="figure" id="figure-1">
+  <p class="figure">Figure 1: The Chunky GUI</p>
+  <div class="figureimgcontainer">
+    <a href="../../../../img/reference/user_interface/chunky/overview-stable.png">
+      <img class="figure" src="../../../../img/reference/user_interface/chunky/overview-stable.png" alt="Chunky GUI">
+    </a>
+  </div>
+</div>
+
+{% endif %}
+
+{% if extra.chunky >= 2_05_00 %}
+
+The GUI of Chunky is built using JavaFX and is separated into two main resizable control panels. These panels have tabs which contain additional controls. The control panel on the left contains render controls, and the panel on the right contains the world map view and the render preview. Scene management controls are at the top of the window, and information about render progress is displayed at the bottom of the window, along with a render progress bar.
+
+<div class="figure" id="figure-1">
+  <p class="figure">Figure 1: The Chunky GUI</p>
+  <div class="figureimgcontainer">
+    <a href="../../../../img/reference/user_interface/chunky/overview-snapshot.png">
+      <img class="figure" src="../../../../img/reference/user_interface/chunky/overview-snapshot.png" alt="Chunky GUI">
+    </a>
+  </div>
+</div>
+
+{% endif %}
+
+--8<-- "includes/abbreviations.md"
```

Comment:
Next -> File (Scene Management)

404
http://127.0.0.1:8000/docs/reference/user_interface/chunky/menu_bar/file/

Next article in menu is Scene Management.